catalog/app/containers/Bucket/PackageDialog/FilesInput.tsx

```diff
@@ -194,7 +195,13 @@ interface DispatchFilesAction {
   (action: FilesAction): void
 }
 
-type FilesEntryState = 'deleted' | 'modified' | 'unchanged' | 'hashing' | 'added'
+type FilesEntryState =
+  | 'deleted'
+  | 'modified'
+  | 'unchanged'
+  | 'hashing'
+  | 'added'
+  | 'invalid'
```

Comment:
to me "invalid" property seems orthogonal to the state

Reply:
since, for example, both a modified and an added entry can be invalid in the same time

Reply:
I agree, and it was my first attempt to implement invalid micro-state for any other state (added/existing), but it was too difficult

Now, I think it's not that hard (probably, because I forgot complexities), and I would like to fix it as a tech/UX dept in a separate task